usim.py

```diff
@@ -384,7 +384,8 @@ def monitor_sub(q, index):
 	t0 = time.time()
 	log('Started main thread %d' % (index+1))
 	while True:
-		log("Restarting loop", console_only=True)
+		#log("Restarting loop", console_only=True)
```

Comment:
Yeah, you're right. The rest of the code includes at least eight other commented `log()` calls:

    $ grep -rc \#log *.py
    rlogin.py:0
    usimb.py:5
    usim.py:3


Sorry for the noise.